App/localization/languages/en.json

```diff
@@ -43,7 +43,8 @@
   "details_header_latest_update_label": "Latest Update:",
   "details_header_latest_update_ago": "ago",
   "details_header_primary_pollutant_label": "Primary Pollutant:",
-  "details_distance_label": "AQI Station: {{distanceToStation}}km away",
+  "details_distance_label": "AQI Station: {{distanceToStation}}{{distanceUnit}} away",
+  "distance_unit": "mi",
```

Comment:
Instead of doing `distanceUnit === 'mi' ? 'mile' : 'km'`, I would recommend to put here:
```
"distance_unit": "mi",
"distance_unit_long": "mile",
```

and use `distance_unit_long` in Details.tsx and Header.tsx

Reply:
Needed to add type to `stations.ts`.  Also labeled it `haversine_distance_unit` as I don't believe `distance_unit_long` doesn't make sense given haversine module's unit value (`km` | `mile` | `meter` | `nmi`). Also shouldn't be mistakenly be used in UX.